pandas/tests/indexes/period/test_period_range.py

```diff
@@ -51,6 +51,9 @@ def test_construction_from_period(self):
         result = period_range(start=start, end=end, freq="Q", name="foo")
         tm.assert_index_equal(result, expected)
 
+        result = period_range(start=start, end=end, freq="Q", name="foo")
+        tm.assert_equal(result, result.values)
```

Comment:
can you follow the OP, e.g. 
```
idx = period_range(...)
result = idx == idx.values
assert result.all()
```

don't object to the above check as well

Reply:
Thanks , hope this looks ok?